### ISSUE TITLE
Link to the 'bastion of the Turbofish'

### ISSUE THREAD

**Issue by Qqwy** [jplatte/turbo.fish#15]
it might make sense to link to https://github.com/rust-lang/rust/blob/master/src/test/ui/bastion-of-the-turbofish.rs

as it was an important example to show why it would be non-trivial to remove the Turbofish from the language.

And it is also just a fun read which I think will combine well with the rest of https://turbo.fish . :slightly_smiling_face:

**Comment by jplatte** [comment]
Right, I meant to add some sort of about page a long time ago but never got to it. I'm a bit hesitant to add more links to the bottom left section.

**Comment by jplatte** [comment]
I'd be happy to see a PR that changes the first `what?` link to go to an explanatory sub-page, maybe just replicating `README.md` (which could also use a mention of the bastion of the turbofish) and removing the `what??` link.

**Comment by Qqwy** [comment]
I'd love to help, but I do not think I'm the right person to write that explanatory sub-page as I am relatively new to the Rust community myself, so I am not aware of the exact history. (Which is of course what brought me to https://turbo.fish in the first place)

**Comment by jplatte** [comment]
Well, content could just be the same as the readme. But maybe I'll get to it myself in the coming days.
